doc/releases/changelog-0.35.0.md

```diff
@@ -185,20 +311,23 @@
   now strictly faster for jax.
   [(#4963)](https://github.com/PennyLaneAI/pennylane/pull/4963)
 
+* PennyLane can now use lightning provided VJPs by selecting `device_vjp=True` on the QNode.
+  [(#4914)](https://github.com/PennyLaneAI/pennylane/pull/4914)
+
 * `device_vjp` can now be used with normal Tensorflow. Support has not yet been added
```

Comment:
🤔

Reply:
This is one of those improvements-section ones where the user facing implication is perhaps not so clear until a future release :sweat_smile: In 0.36 we'll hopefully have `device_vjp` enabled by default (main improvement should be big speedups for cost functions with one or just a few dimensions). We'll advertise it much more prominently then.

Reply:
I would remove or rephrase 'normal TensorFlow' here lol.

But yes, I think most of these bullet points can likely be rephrased to focus on upcoming user implications (or even current user implications, because they don't specify you can access this already using `device_vjps=True` in the QNode